README.md

```diff
@@ -1,29 +1,45 @@
 # Framework
 Common framework for all cp3-llbb analyses
 
-## Setup instructions
+*Please note:*
+* The instructions are for the UCLouvain ingrid SLC6 cluster (to access SAMADhi)
+* You need the proper username and password to access SAMADhi :) If you don't know what this is about, ask around
+* The current state of the art mini-AOD documentation can be found [here](https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookMiniAOD2015)
 
-```bash
 
+
+## First time setup instructions
+
+```bash
+source /nfs/soft/grid/ui_sl6/setup/grid-env.sh
+source /cvmfs/cms.cern.ch/cmsset_default.sh
 export SCRAM_ARCH=slc6_amd64_gcc491
-cmsrel CMSSW_7_4_4
-cd CMSSW_7_4_4/src
+cmsrel CMSSW_7_4_5
+cd CMSSW_7_4_5/src
 cmsenv
 
 git cms-init
 cd ${CMSSW_BASE}/src 
 
+# Electron ID as from [June 28th 2015 EGamma hypernews](https://hypernews.cern.ch/HyperNews/CMS/get/egamma/1589.html)
+git cms-merge-topic ikrav:egm_id_74X_v2
```

Comment:
well, the [twiki](https://twiki.cern.ch/twiki/bin/view/CMS/MultivariateElectronIdentificationRun2#Recipes_for_regular_users_common) indicates somewhat otherwise... in any case I tried running without and end up with:

```
Added 'egmGsfElectronIDs' to process definition (MiniAOD format)!
----- Begin Fatal Exception 30-Jun-2015 18:15:34 CEST-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named TestConfiguration.py
Exception Message:
python encountered the error: <type 'exceptions.ImportError'>
No module named mvaElectronID_PHYS14_PU20bx25_nonTrig_V1_cff
----- End Fatal Exception -------------------------------------------------
```